api/app/services/training/unified_faq_extractor.py

```
@@ -674,17 +694,37 @@ def get_anon_name(author: str) -> str:
+
             if citation:
                 cited_author = citation.get("author", "unknown")
                 cited_text = citation.get("text", "")[:50]
                 anon_cited = get_anon_name(cited_author)
-                line += f' (replying to {anon_cited}: "{cited_text}...")'
-
-            reply_to = msg.get("reply_to")
-            if reply_to:
-                line += f" (reply to: {reply_to})"
+                cited_msg_num = None
+                for mid, num in id_to_msg_number.items():
```

Ruff (0.15.10)

[warning] Loop control variable `mid` not used within loop body

(B007)